DiscordBot.js

```diff
@@ -15,45 +15,216 @@ class DiscordBot {
     
         this.client.on('message', msg => {
 
-            let prefix = "§";
-            let channelFilePath = "config/discordchannels.json";
+            let prefix = "$";
+
+            const channelsFilePath  = "config/channels.json";
+            const channelsFile      = JSON.parse(fs.readFileSync(channelsFilePath));
+            const coursesFilePath   = "config/courses.json";
+            const coursesFile       = JSON.parse(fs.readFileSync(coursesFilePath));
+
+            let respondContent = "";
+            
+            const addColor      = "#00ff00";
+            const deleteColor   = "#ff0000";
+            const messageColor  = "#555555";
 
             if (msg.content.startsWith(prefix)) {
+                this.botLog(msg);
+                let args = msg.content.split(" "); //split in arguments
                 switch(true) {
```

Comment:
Dieser gesamte switch Ausdruck sollte in eine config json ausgelagert werden und ihre einzelnen cases in functions ausgelagert werden. Das würde dem ganzen mehr dynamische und allgemein durchsichte Struktur geben.